src/button-layout.tsx

```diff
@@ -59,7 +59,7 @@ const ButtonLayout: React.FC<ButtonLayoutProps> = ({
 
     const linkContainer = link ? (
         <div
-            className={classnames(numberOfButtons > 1 ? styles.linkWithTwoButtons : styles.link)}
+            className={classnames(numberOfButtons !== 1 ? styles.linkWithTwoButtons : styles.link)}
```

Comment:
the `styles.linkWithTwoButtons` name doesn't make sense now. I'd rename it to `styles.linkInNewLine` or something similar

Reply:
fixed